docs/PLUGIN_CATALOG.md

```diff
@@ -10,21 +10,22 @@ This is the master catalog of plugins for Snap. While the Snap framework is hard
 ## All Plugins
 This file is automatically generated. If you would like to add to the plugin list, [add your plugin to this list](plugins.yml) and it will be added (usually within 24 hours).
 
-### Collectors (61)
+### Collectors (62)
 
 | Name | Maintainer | Description | CI | Download |
 |------|------------|-------------|----|----------|
-| [Apache](https://github.com/intelsdi-x/snap-plugin-collector-apache) | [Intel](https://github.com/intelsdi-x) | Collects Apache webserver mod_status metrics | [![Build Status](https://travis-ci.org/intelsdi-x/snap-plugin-collector-apache.svg?branch=master)](https://travis-ci.org/intelsdi-x/snap-plugin-collector-apache) [![Build Status](https://ci.snap-telemetry.io/job/snap-plugin-collector-apache-ec2-periodic/badge/icon)](https://ci.snap-telemetry.io/job/snap-plugin-collector-apache-ec2-periodic/) | [release](https://github.com/intelsdi-x/snap-plugin-collector-apache/releases) |
```

Comment:
Oh I see now that Iza manually edited this file to add large test status in 05da8279b74647a63249da79437c6c0cce4e2e3d. This file is not intended to be manually edited. I can still see if we can get large status into this PR, but you cheated by not using our toolchain @IzabellaRaulin ;)